src/gitops/components/imageupdater/ImageUpdaterList.tsx

```
@@ -0,0 +1,458 @@
+  const effectiveNamespace = listAllNamespaces ? null : namespace;
+  const [imageUpdaters, loaded, loadError] = useK8sWatchResource<K8sResourceCommon[]>({
+    isList: true,
+    groupVersionKind: modelToGroupVersionKind(ImageUpdaterModel),
+    namespaced: !listAllNamespaces,
+    namespace: effectiveNamespace,
+  });
+
+  const columnSortConfig = React.useMemo(() => {
+    return [
+      'name',
+      ...(!listAllNamespaces || !effectiveNamespace || effectiveNamespace === '' ? ['namespace'] : []),
```

ESLint

[error] Replace `·?·['namespace']` with `⏎········?·['namespace']⏎·······`

(prettier/prettier)